fix(vue): Remove outdated Vue SDK options (#13192)
getsentry/sentry-docs: docs/platforms/javascript/common/configuration/options.mdx

</SdkOption>

<PlatformSection supported={["javascript.vue", "javascript.nuxt"]}>

> <SdkOption name="attachProps" type='boolean' defaultValue='true'>

Reviewer: ```suggestion
<SdkOption name="attachProps" type='boolean' defaultValue='true' categorySupported={['server']}>
```

Since for nuxt this is only client side, right?

Author: Oh nice, I didn't know that this is possible! 

> Since for nuxt this is only client side, right?

But it should be `categorySupported={['browser']}` then right?

Author: fixed in [388d8bd](https://github.com/getsentry/sentry-docs/pull/13192/commits/388d8bdc3ad70d5c74283b72122f89b1e2a3bb59)

When enabled, all props of the Vue component where the error was thrown in are attached to the event.

</SdkOption>

> <SdkOption name="attachProps" type='boolean' defaultValue='true'>

Reviewer: ```suggestion
<SdkOption name="attachProps" type='boolean' defaultValue='true' categorySupported={['server']}>
```

Author: [388d8bd](https://github.com/getsentry/sentry-docs/pull/13192/commits/388d8bdc3ad70d5c74283b72122f89b1e2a3bb59)

</SdkOption>

<PlatformSection supported={["javascript.vue", "javascript.nuxt"]}>

<SdkOption name="attachProps" type='boolean' defaultValue='true' categorySupported={['browser']}>

When enabled, all props of the Vue component where the error was thrown in are attached to the event.

</SdkOption>

<SdkOption name="attachErrorHandler" type='boolean' defaultValue='true' categorySupported={['browser']}>
  By default, the SDK attaches a [Vue error handler](https://vuejs.org/api/application.html#app-config-errorhandler)
  to capture Vue exceptions and report them to Sentry.
  When `attachErrorHandler` is set to `false`, automatic error reporting is disabled.

  Usually, this option should stay enabled, unless you want to set up Sentry error reporting yourself.

>   For example, the Sentry Nuxt SDK does not attach an error handler as it's using the error hooks provided by Nuxt.

Reviewer 2: ```suggestion
  For example, the Sentry Nuxt SDK does not attach an error handler as it's capturing errors through the error hooks provided by Nuxt.
```
